.github/workflows/pr.yml

```diff
@@ -85,16 +83,15 @@ jobs:
         with:
           name: go-binaries
           path: bin/
-      - name: Publish Pulumi Java SDK
-        run: make sdk.publish
-        # See Note 1.
-        if: |
-          github.event_name == 'repository_dispatch'
-          || (
-                github.event.pull_request.head.repo.full_name == github.repository
-                && ! startsWith(github.ref_name, 'dependabot/')
-             )
-          || github.ref_name == 'main'
+      - name: Publish Pulumi Java SDK to Maven Local
+        uses: gradle/gradle-build-action@9b814496b50909128c6a52622b416c5ffa04db49
```

Comment:
Why this commit? https://github.com/gradle/gradle-build-action/commit/9b814496b50909128c6a52622b416c5ffa04db49

Would just v2.2.2 work?

Reply:
Eh, copied from elsewhere thoughtlessly. It should work yes. We should probably clean up and use a consistent version across all the build files (in a separate PR).